.teamcity/_self/PluginBaseBuild.kt

```diff
@@ -59,6 +59,24 @@ open class PluginBaseBuild : Template({
 		bashNodeScript {
 			name = "Prepare environment"
 			scriptContent = """
+				# Merge the trunk branch first. This way, our builds and tests
+				# include the latest merged version of the plugin being built.
+				# Otherwise, we can get into a situation where the current plugin
+				# build appears "different", but that's just because it's older.
+				if [[ "%teamcity.build.branch.is_default%" != "true" ]] ; then
+					# git operations will fail if no user is set.
+					git config --local user.email "tcbuildagent@example.com"
```

Comment:
Shouldn't we change this to a real mailbox?

Reply:
I guess I'm not sure. `@example.com` is meant to be used for fake emails that don't go anywhere. The merge commit isn't going to leave the local server, so it's arbitrary from my point of view. In other words, the only reason we set a user is because git forces us to, not because the user is actually used for anything meaningful